source/uri-options/tests/connection-options.yml

```diff
@@ -104,3 +104,46 @@ tests:
         hosts: ~
         auth: ~
         options: {}
+
+    -
+      description: directConnection=true
+      uri: "mongodb://example.com/?directConnection=true"
+      valid: true
+      warning: false
+      hosts: ~
+      auth: ~
+      options:
+          directConnection: true
+    -
+      description: directConnection=true with multiple seeds
+      uri: "mongodb://example1.com,example2.com/?directConnection=true"
+      valid: false
+      warning: false
+      hosts: ~
+      auth: ~
+    -
+      description: directConnection=false
+      uri: "mongodb://example.com/?directConnection=false"
+      valid: true
+      warning: false
+      hosts: [example.com]
```

Comment:
According to the tests README, this field will [always be null](https://github.com/mongodb/specifications/blob/master/source/uri-options/tests/README.rst#format) and is just included so that the connection string test runner can be reused for these tests. If we do want to keep it around, it needs to match the format of the connection string tests (described [here](https://github.com/mongodb/specifications/tree/master/source/connection-string/tests#format)). I think rather than trying to do that, it's probably simpler to just leave this null, since we're not testing host parsing here anyways.

Reply:
@p can you respond please?

Reply:
Removed the hosts.